compose-guard/core/src/main/kotlin/com/joetr/compose/guard/core/ComposeMetricsContentProvider.kt

```diff
@@ -26,9 +26,9 @@ package com.joetr.compose.guard.core
 /**
  * Provides content of a composable report and metrics
  */
-public class ComposeMetricsContentProvider(private val fileProvider: ComposeCompilerRawReportProvider) {
-    public val briefStatisticsContents: List<String> get() = fileProvider.briefStatisticsJsonFiles.map { it.readText() }
-    public val detailedStatisticsCsvRows: List<String> get() = fileProvider.detailedStatisticsCsvFiles.flatMap { it.readLines() }
-    public val composablesReportContents: String get() = fileProvider.composableReportFiles.joinToString(separator = "\n") { it.readText() }
-    public val classesReportContents: String get() = fileProvider.classesReportFiles.joinToString(separator = "\n") { it.readText() }
+internal class ComposeMetricsContentProvider(private val fileProvider: ComposeCompilerRawReportProvider) {
```

Comment:
@JJSwigut I missed it on the original PR, but when enabling explicitApi, you want to favor visibility modifiers in the order private, internal, protected, public. for classes that aren't exposed outside of the module, we want to keep their visibility modifiers internal to the module.

Reply:
when we make things unnecessarily public, they become part of the public API and we have to be aware of how they are consumed by downstream consumers for every release. we help check this by using the compatibility validator tool https://github.com/Kotlin/binary-compatibility-validator